app/main/views/frameworks.py

```diff
@@ -293,6 +294,61 @@ def framework_submission_lots(framework_slug):
     ), 200
 
 
+@main.route('/frameworks/<framework_slug>/submissions/service-type', methods=['GET', 'POST'])
+@login_required
+@EnsureApplicationCompanyDetailsHaveBeenConfirmed(data_api_client)
+def choose_draft_service_lot(framework_slug):
+    framework = get_framework_or_404(data_api_client, framework_slug)
+
+    if framework['status'] not in ["open", "pending", "standstill"]:
+        abort(404)
+
+    errors = {}
+    status_code = 200
+
+    lot_question = {
+        option["value"]: option
+        for option in ContentQuestion(content_loader.get_question(framework_slug, 'services', 'lot')).get('options')
+    }
+
+    lots = [
+        {
+            "text": lot_question[lot['slug']]['label'] if framework["status"] == "open" else lot["name"],
```

Comment:
I find the "Apply to provide [type of team]" options a bit confusing, is there a way we can change the content so it reads more like an answer to the question @NoraGDS?

![Screenshot of form, "What type of service do you want to add?", with options "Apply to provide teams for digital outcomes", "Apply to provide individual digital specialists"](https://user-images.githubusercontent.com/22524634/110331317-dd938300-8016-11eb-9c50-752b287ad99b.png)

Reply:
There's already a pull request open to change the hint text https://github.com/alphagov/digitalmarketplace-frameworks/pull/674/files

Reply:
@domoscargin Oh I think maybe Nora did suggest that change already and it got missed from the PR? https://trello.com/c/r8CxfKTO/820-5-replace-choose-a-lot-page-in-create-a-service-journey#comment-603f9fd5d68a9d50a6176e79

Reply:
D'oh, good catch - I've updated the Frameworks PR: https://github.com/alphagov/digitalmarketplace-frameworks/pull/674/files